src/index.js

```diff
@@ -4,6 +4,7 @@ import schemas from './schemas';
 import Ajv from 'ajv';
 import deprecate from 'util-deprecate';
 import equals from 'fast-deep-equal';
+import RE2 from 're2';
```

Comment:
This would not work in the browser, since RE2 contains native code. Maybe we could fall back on the built in regex engine if RE2 isn't available?